prow/config.gen.yaml

```diff
@@ -2468,6 +2468,55 @@ presubmits:
       - emptyDir: {}
         name: varlibdocker
 
+  - name: istio-push-images-2.3
+    trigger: (?m)^/test( | .* )push\-images,?($|\s.*)
+    rerun_command: /test push-images
```

Comment:
@jwendell Do you know if we can change this to `/push-images` or something like that (without `/test`).

Reply:
ok, let's try.